src/utils/utils.tsx

```diff
@@ -0,0 +1,51 @@
+import type { Dispatch, SetStateAction } from 'react';
+import { AxiosError, AxiosResponse } from 'axios';
+
+export type SetterFunction<T> = Dispatch<SetStateAction<T>>;
+export type UserKindType = 'brand' | 'creator' | '';
+export type Callbacks<T = any> = {
+  success?: (value?: T) => void;
+  error?: (value?: string) => void;
+  finally?: () => void;
+};
+
+export const isPasswordValid = (value: string) => {
+  const passwordRegex = new RegExp(
+    '^(((?=.*[a-z])(?=.*[A-Z]))|((?=.*[a-z])(?=.*[0-9]))|((?=.*[A-Z])(?=.*[0-9])))(?=.{8,})'
+  );
+  return passwordRegex.test(value);
+};
+
+export const isEmailValid = (value: string) => {
+  if (value.length > 50) return false;
+  const emailRegex = /^.+@.+$/;
+
+  return emailRegex.test(value);
+};
+
+export const manageResponse = (
+  response: AxiosResponse<any>,
```

Comment:
Agustín sent me (this)[https://stackoverflow.com/questions/65678916/typescript-how-to-change-type-of-axios-response-when-modified-interceptor-to-r] though I honestly don't understand how to apply it to this case, it might help for something though